### DISCUSSION TITLE
Entrance Animations: Different Timings and Animations in Responsive Mode

### DISCUSSION THREAD

**Discussion by mr-sparks** [elementor/elementor#27431] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When using entrance animations on an Elementor page, there is no way to change the delay specific for different mobile breakpoints. This would be important since on mobile view there might be several elements hidden, which will take animations that come after that, longer than on the desktop, and break the feeling of the animation.

### Describe the solution you'd like.

It would be great if, when switching into responsive mode, it it would be possible to give different timeout / delay fore mobile / tablet etc.

### Describe alternatives you've considered.

There are no alternatives right now.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by onan** [comment]
Bump. This would be very useful.
